### ISSUE TITLE
Create assortment pin list in modal, and pin assortment link

### ISSUE THREAD

**Issue by stretch1414** [PinsterTeam/Frontend#33]
We need to design a flow for pins and assortments to be associated with each other.  I'm thinking that the bottom of the modal will have a list of Pins for an Assortment, and an Assortment URL (per Andrew) for the Pin.  Would like to work this out with you @markmoser .  I'm thinking that the Pin list would be almost like the ListItems we already have, therefore popping a new modal on click and hiding/destroying the old modal.  You will probably have a better idea, but that is just what I thought up.

**Comment by stretch1414** [comment]
Yeah that's mostly all I was thinking.  Didn't know how easy/hard it would be to switch to a new modal.

**Comment by markmoser** [comment]
Unless it looks/works like the "pin" modal, I would model off the form modal. Just use the same wrapper class names and well refactor for a more generic name later.

**Comment by stretch1414** [comment]
I kind of thought it would be very similar to the Pin modal, since the data model from the database is very similar between the two "Collectables".

**Comment by markmoser** [comment]
Maybe this should use the same ul/pattern we have for "add to collection"

**Comment by stretch1414** [comment]
Maybe.  But a Pin can only be a part of one Set/Assortment.
